<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphicEQFilterGUI</class>
 <widget class="QWidget" name="GraphicEQFilterGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>124</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Graphic EQ:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton15">
       <property name="text">
        <string>15-band</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton31">
       <property name="text">
        <string>31-band</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButtonVar">
       <property name="text">
        <string>variable</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="GraphicEQFilterGUIView" name="graphicsView">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="renderHints">
      <set>QPainter::RenderHint::Antialiasing|QPainter::RenderHint::TextAntialiasing</set>
     </property>
     <property name="dragMode">
      <enum>QGraphicsView::DragMode::RubberBandDrag</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QTableWidget" name="tableWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="palette">
        <palette>
         <active>
          <colorrole role="Highlight">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>51</red>
             <green>153</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="Highlight">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>122</red>
             <green>189</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="Highlight">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>51</red>
             <green>153</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::SizeAdjustPolicy::AdjustToContents</enum>
       </property>
       <property name="horizontalScrollMode">
        <enum>QAbstractItemView::ScrollMode::ScrollPerPixel</enum>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="columnCount">
        <number>2</number>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>10</number>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>10</number>
       </attribute>
       <attribute name="horizontalHeaderHighlightSections">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderMinimumSectionSize">
        <number>23</number>
       </attribute>
       <attribute name="verticalHeaderDefaultSectionSize">
        <number>23</number>
       </attribute>
       <column>
        <property name="text">
         <string>Freq.</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Gain</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <widget class="QToolBar" name="toolBar"/>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionImport">
   <property name="icon">
    <iconset resource="../Editor.qrc">
     <normaloff>:/icons/document-open.ico</normaloff>:/icons/document-open.ico</iconset>
   </property>
   <property name="text">
    <string>Import</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="icon">
    <iconset resource="../Editor.qrc">
     <normaloff>:/icons/document-save-as.ico</normaloff>:/icons/document-save-as.ico</iconset>
   </property>
   <property name="text">
    <string>Export</string>
   </property>
  </action>
  <action name="actionInvertResponse">
   <property name="icon">
    <iconset resource="../Editor.qrc">
     <normaloff>:/icons/invert_response.svg</normaloff>:/icons/invert_response.svg</iconset>
   </property>
   <property name="text">
    <string>Invert response</string>
   </property>
  </action>
  <action name="actionNormalizeResponse">
   <property name="icon">
    <iconset resource="../Editor.qrc">
     <normaloff>:/icons/normalize_response.svg</normaloff>:/icons/normalize_response.svg</iconset>
   </property>
   <property name="text">
    <string>Normalize response</string>
   </property>
  </action>
  <action name="actionResetResponse">
   <property name="icon">
    <iconset resource="../Editor.qrc">
     <normaloff>:/icons/reset_response.svg</normaloff>:/icons/reset_response.svg</iconset>
   </property>
   <property name="text">
    <string>Reset response</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphicEQFilterGUIView</class>
   <extends>QGraphicsView</extends>
   <header>GraphicEQFilterGUIView.h</header>
   <slots>
    <signal>itemMoved()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../Editor.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>updateItemList()</slot>
 </slots>
</ui>
